nautobot/core/views/generic.py

```diff
@@ -255,17 +254,17 @@ def get(self, request):
                 self.queryset = self.queryset.none()
 
             display_filter_params = [
-                check_filter_for_display(filterset_filters, field_name, values)
+                check_filter_for_display(filterset.filters, field_name, values)
```

Comment:
Same change needed in `nautobot/core/views/renderers.py`? In `get_context()`?

Reply:
Ugh... Just now realized the instantiation of the filterset is in a mixin...

So this: https://github.com/nautobot/nautobot/blob/372d01e47a46838c5816ffc04758934bb162351e/nautobot/core/views/renderers.py#L160


Does this under the hood: https://github.com/nautobot/nautobot/blob/372d01e47a46838c5816ffc04758934bb162351e/nautobot/core/views/mixins.py#L454